R_scripts/tree_proximity.R

```diff
@@ -30,11 +32,21 @@ cat("Joining datasets...\n")
 collar_to_tree_prox <- select(proxDat, -Date) %>% 
   left_join(treeDat, by = c("Site", "Plot", "Tag"), na_matches = "never") %>%
   mutate(BA_sqm = (DBH_cm / 100 / 2) ^ 2 * pi)  # from DBH (cm) to area (m2)
+write_csv(collar_to_tree_prox, "../inventory_data/collar_to_tree_prox.csv")
+
+# Replace DBH with recorded value for non-tagged trees
+for (i in nrow(collar_to_tree_prox)) {
```

Comment:
`for(i in seq_len(nrow(collar_to_tree_prox)) {`? The code above will only execute once.

Reply:
You actually don't even need a loop at all:
```R
no_tags <- is.na(collar_to_tree_prox$Tag)
collar_to_tree_prox$DBH_cm[no_tags] <- collar_to_tree_prox$No_tag_DBH[no_tags]
collar_to_tree_prox$Species[no_tags] <- collar_to_tree_prox$No_tag_species[no_tags]
```